tests/e2e/specs/shopper/tax.test.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Internal dependencies
+ */
+import { shopper } from '../../../utils';
+import { sleep } from '../../utils';
+import { Taxes, Products } from '../../fixtures/fixture-data';
+import {
+	getExpectedTaxes,
+	getTaxesFromCurrentPage,
+	getTaxesFromOrderSummaryPage,
+	showTaxes,
+} from '../../../utils/taxes';
+
+const taxRates = Taxes();
+const productWooSingle1 = Products().find(
+	( prod ) => prod.name === 'Woo Single #1'
+);
+
+jest.setTimeout( 120000 );
```

Comment:
why is this needed?

Reply:
These tests are timing out, but only on CI. This is just me trying to increase the timeout to see if it fixes it (it didn't, it doesn't seem to increase the timeout at all). Weirdly this doesn't work inside a test or inside any of the `before` hooks which is why I've tried putting it out here

Reply:
I had a test timing out for me locally. We need to increase the general setting, as we will have a lot of _longer_ tests. Can you try to only add this new value [here](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/2bfecbcf4ac26ea3af89c5929864ab119bd3263b/tests/e2e/config/jest.setup.js#L13-L14) and see the result.

Reply:
Yep this worked, thanks :)